features/admin.logs.v1/components/remote-logging-config-form.tsx

```diff
@@ -0,0 +1,461 @@
+                        <Skeleton variant="rectangular" height={ 7 } width="90%"/>
+                        <Skeleton variant="rectangular" height={ 7 }/>
+                    </Stack>
+                </div>
+            </Card>
+        );
+    }
+
+    function handleTestNow() {
+        setTestNow(true);
+        mutateTest();
+    }
+
+    return (
+        <>
+            <Card className="remote-logging-content">
+                <div className="remote-logging-form">
+                    <Grid xs={ 12 } md={ 8 } lg={ 4 }>
+                        <FinalForm
+                            onSubmit={ (values: RemoteLogPublishingConfigurationInterface) =>
+                                handleRemoteLoggingConfigUpdate(values) }
+                            data-componentid={ `${componentId}-${logType}` }
+                            initialValues={ logConfigData }
+                            render={ ({ handleSubmit }: FormRenderProps) => (
+                                <form onSubmit={ handleSubmit }>
+                                    <FinalFormField
+                                        key="url"
+                                        FormControlProps={ {
+                                            margin: "dense"
+                                        } }
+                                        width={ 16 }
+                                        className="text-field-containe"
+                                        ariaLabel="url"
+                                        required={ true }
+                                        data-componentid={ `${componentId}-url` }
+                                        name="url"
+                                        type="text"
+                                        label={ t("console:manage.features.serverConfigs." +
+                                            "remoteLogPublishing.fields.remoteURL.label") }
+                                        placeholder={ t("console:manage.features.serverConfigs." +
+                                            "remoteLogPublishing.fields.remoteURL.placeholder") }
+                                        validate={
+                                            (value: FormValue) => {
+                                                if (!value) {
+                                                    return t("console:manage.features.serverConfigs." +
+                                                        "remoteLogPublishing.fields.remoteURL.error.required");
+                                                } else if (!FormValidation.url(value.toString())) {
+                                                    return t("console:manage.features.serverConfigs." +
+                                                        "remoteLogPublishing.fields.remoteURL.error.invalid");
+                                                }
+                                            }
+                                        }
```

Comment:
I'd suggest to use form level validation rather than field level. Is there any blocker for that?

Ref: https://github.com/wso2/identity-apps/blob/master/features/admin.remote-userstores.v1/components/create/configurations-form.tsx#L122